cpuset/cpuset.go

```diff
@@ -0,0 +1,254 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package cpuset represents a collection of CPUs in a 'set' data structure.
```

Comment:
Maybe also worth to mention where the file got copied over for history tracking?

Reply:
This is k/utils. I am ok to make changes here after https://github.com/kubernetes/kubernetes/pull/115359 merged

Reply:
Added an origin comment to cpuset.go only